### PR TITLE
TxTypes for web3.js Ext

### PR DESCRIPTION
**Pull request by nohkwak**
- From ethers.js EXT, TxType Examples for Basic, FeeDelegation and PartialFeeDelegation are ported, tested, and succeeded. 
(The Partial FeeDelegation part was completed more easily than I expected, so it was added.)

### CODE REVIEW

File: web3js-ext/src/web3/klaytn_tx.ts [hunk 1]
```diff
@@ -116,9 +123,10 @@ export class KlaytnTx extends LegacyTransaction {
     this._klaytnType = toNumber(txData.type) as number;
     this.from = txData.from;
     this.chainId = txData.chainId;
+    // @ts-ignore
+    this.key = txData.key;
```
Comment (blukat29) [inline web3js-ext/src/web3/klaytn_tx.ts:127]: Define 'key' in KlaytnTxData type at L10, then remove ts-ignore.

File: web3js-ext/src/web3/klaytn_tx.ts [hunk 2]
```diff
@@ -116,9 +123,10 @@ export class KlaytnTx extends LegacyTransaction {
...
 
-    // A readonly CoreKlaytnTx object
-    this.klaytnTxData = KlaytnTxFactory.fromObject({
+    let initTxData = {
```
Comment (blukat29) [inline web3js-ext/src/web3/klaytn_tx.ts:129]: Name it klaytnTxObject

File: web3js-ext/src/web3/klaytn_tx.ts [hunk 3]
```diff
@@ -131,7 +139,18 @@ export class KlaytnTx extends LegacyTransaction {
       data:     bytesToHex(this.data),
       input:    bytesToHex(this.data),
       chainId:  this.chainId ? toHex(this.chainId) : undefined,
-    });
+      humanReadable: false, 
+      codeFormat: 0x00,
+      key: this.key,
+    }; 
+
+    if ( txData.type == 0x28 ) {
```
Comment (blukat29) [inline web3js-ext/src/web3/klaytn_tx.ts:147]: Use the TxType enum please

**Review by blukat29 — COMMENTED** [review]
If you haven't modified any code in FeeDel & PartialFeeDel examples, then remove those files from this PR for clarity.

File: web3js-ext/src/web3/klaytn_tx.ts (hunk 1 above)
Comment (nohkwak) [reply web3js-ext/src/web3/klaytn_tx.ts]: Yes I did https://github.com/klaytn/web3klaytn/pull/507/commits/a45019917a79438eeefc27f034697c8daef05bfd

File: web3js-ext/src/web3/klaytn_tx.ts (hunk 2 above)
Comment (nohkwak) [reply web3js-ext/src/web3/klaytn_tx.ts]: Thanks https://github.com/klaytn/web3klaytn/pull/507/commits/854b25be74d13e7dd6d7b301849433e61e05e86b

File: web3js-ext/src/web3/klaytn_tx.ts (hunk 3 above)
Comment (nohkwak) [reply web3js-ext/src/web3/klaytn_tx.ts]: Good, I checked https://github.com/klaytn/web3klaytn/pull/507/commits/076027bbf9c1532934985aa561100b15b6df0c54

**Review by blukat29 — APPROVED** [review]

**Pull request merged**